src/bootstrap/build/native.rs

```diff
@@ -63,7 +63,7 @@ pub fn llvm(build: &Build, target: &str) {
        .out_dir(&dst)
        .profile(if build.config.llvm_optimize {"Release"} else {"Debug"})
        .define("LLVM_ENABLE_ASSERTIONS", assertions)
-       .define("LLVM_TARGETS_TO_BUILD", "X86;ARM;AArch64;Mips;PowerPC")
+       .define("LLVM_TARGETS_TO_BUILD", "X86;ARM;AArch64;Mips;PowerPC;NVPTX")
```

Comment:
I don't have any precise measurement but:

```
# Nightly of 2016-06-08
$ du -h $(rustc --print sysroot)/lib
28K     /home/japaric/.multirust/toolchains/nightly-2016-06-08-x86_64-unknown-linux-gnu/lib/rustlib/etc
147M    /home/japaric/.multirust/toolchains/nightly-2016-06-08-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib
147M    /home/japaric/.multirust/toolchains/nightly-2016-06-08-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu
147M    /home/japaric/.multirust/toolchains/nightly-2016-06-08-x86_64-unknown-linux-gnu/lib/rustlib
254M    /home/japaric/.multirust/toolchains/nightly-2016-06-08-x86_64-unknown-linux-gnu/lib

# This PR on top of that nightly (Note: stage1)
$ du -h build/x86_64-unknown-linux-gnu/stage1/lib
155M    build/x86_64-unknown-linux-gnu/stage1/lib/rustlib/x86_64-unknown-linux-gnu/lib
155M    build/x86_64-unknown-linux-gnu/stage1/lib/rustlib/x86_64-unknown-linux-gnu
155M    build/x86_64-unknown-linux-gnu/stage1/lib/rustlib
253M    build/x86_64-unknown-linux-gnu/stage1/lib
```

Also the LLVM static libraries are rather small:

```
$ ls -hs build/x86_64-unknown-linux-gnu/llvm/lib/*PTX*
160K build/x86_64-unknown-linux-gnu/llvm/lib/libLLVMNVPTXAsmPrinter.a
2.3M build/x86_64-unknown-linux-gnu/llvm/lib/libLLVMNVPTXCodeGen.a
384K build/x86_64-unknown-linux-gnu/llvm/lib/libLLVMNVPTXDesc.a
8.0K build/x86_64-unknown-linux-gnu/llvm/lib/libLLVMNVPTXInfo.a
```

No data on compile times.